src/LinkDotNet.Blog.Web/Features/ShowBlogPost/ShowBlogPostPage.razor

```diff
@@ -72,6 +72,9 @@ else
     [Parameter]
     public string BlogPostId { get; set; }
 
+    [Parameter]
+    public string SearchEngineFriendlyTitle { get; set; }
```

Comment:
I would remove that whole part here. 
The component doesn't do anything with it, anyway.

Reply:
Fair point - yes, that will not work.